packages/runtime/runtime-utils/src/dataStoreHelpers.ts

```diff
@@ -23,6 +23,7 @@ interface IResponseException extends Error {
     message: string;
     code: number;
     stack?: string;
+    underlyingResponseHeaders?: { [key: string]: any; };
```

Comment:
Should we just call them "headers" similar to how IResponse has headers? It is after all a response exception.

Reply:
That's how it was, I changed it last night. Really, handle.get should expose its own shape of error and not expose the fact that internally it's making a request. But I don't want to design that right now. 

I can switch it back to headers if you care. "It's a response exception" - well there's nothing about the actual data that says that, just the TS type. Probably should have errorType but again not touching that here.

Reply:
I don't care that much :-). It's not a big deal as its internal.
Agree with you on handle.get() errors. We should define the error type. Hopefully with the handle work, this will get done. We can create an issue for now.